### DISCUSSION TITLE
Stops at Upgrading macOS/Preparing macOS for installation in jamf Self Service

### DISCUSSION THREAD

**Discussion by jlcpremier** [grahampugh/erase-install#277] [General]
Hi Graham, this is an awesome script and I'm trying to use it with the pkg installer with depnotify uploaded to jamf and triggering the policy via self service.

It runs perfectly fine up to the window to prepare macOS for installation, but then the window closes and self service says the policy has run successfully. I used to use this script back in the day (https://github.com/kc9wwh/macOSUpgrade) and if I remember correctly there needed to be a kill all command for self service included which otherwise would prevent the OS install from running. I'm thinking the same thing applies here. Would you be able to tell me where to input that command in the script, or what else could be causing it to close the window and not launch the OS install?

Thanks!

**Comment by grahampugh** [comment]
There is already a killall in the script. Perhaps it's worth looking at the logs to see why it's not restarting (assuming that you are using the correct command options). 

Feel free to post your command and full log output here to help diagnose. Also state which OS you are running the script, and whether you have any security software that might get in the way.
